docs/source/models.rst

```diff
@@ -394,7 +438,17 @@ To deploy remotely to SageMaker you need to set up your environment and user acc
 To export a custom model to SageMaker, you need a MLflow-compatible Docker image to be available on Amazon ECR.
 MLflow provides a default Docker image definition; however, it is up to you to build the image and upload it to ECR.
 MLflow includes the utility function ``build_and_push_container`` to perform this step. Once built and uploaded, you can use the MLflow
-container for all MLflow models.
+container for all MLflow models. Model webservers deployed using the :py:mod:`mlflow.sagemaker`
+module will accept the following data formats as input, depending on the deployment flavor:
+
```

Comment:
module will accept the following data formats -> module accept the following data formats